src/extras/helpers/WireframeHelper.js

```diff
@@ -6,10 +6,17 @@ THREE.WireframeHelper = function ( object, hex ) {
 
 	var color = ( hex !== undefined ) ? hex : 0xffffff;
 
-	THREE.LineSegments.call( this, new THREE.WireframeGeometry( object.geometry ), new THREE.LineBasicMaterial( { color: color } ) );
+	var geometry = new THREE.WireframeGeometry( object.geometry );
+	var skinning = geometry.skinning === true;
+	if ( skinning && object.skeleton ) {
+		this.skeleton = object.skeleton;
+		this.bindMatrix = object.bindMatrix;
+		this.bindMatrixInverse = object.bindMatrixInverse;
+	}
+	THREE.LineSegments.call( this, geometry, new THREE.LineBasicMaterial( { color: color, skinning: skinning } ) );
 
 	this.matrix = object.matrixWorld;
-	this.matrixAutoUpdate = false;
+	this.matrixAutoUpdate = true;
```

Comment:
You can't do this. It will cause `WireframeHelper` to not work on transformed objects, for example.